### PR TITLE
Configures Issue Templates for the repository.

### LINKED ISSUE #18906: Create an Issue Template for the project
According to the project's [Community Profile](https://github.com/emberjs/ember.js/community), the repository setup can be improved to match the recommended community standards. 

Screen Shot 2020-04-21 at 11 51 57 pm

One of the valuable improvements can be creating an [Issue Template](https://help.github.com/en/github/building-a-strong-community/using-templates-to-encourage-useful-issues-and-pull-requests) to encourage useful issues, simplify reporting and subsequent investigation.

Here are some examples from other repositories:
- [microsoft/TypeScript](https://github.com/microsoft/TypeScript/issues/new/choose)
- [angular/angular](https://github.com/angular/angular/issues/new/choose)
- [aurelia/aurelia](https://github.com/aurelia/aurelia/issues/new/choose)
- [telerik/kendo-angular](https://github.com/telerik/kendo-angular/issues/new/choose)

The template can be potentially used across the whole organization (e.g. for Ember Data and Ember CLI, etc.).

### PR DESCRIPTION
**Pull request by alexeykostevich**
Resolves https://github.com/emberjs/ember.js/issues/18906.

Adds [Issue Templates](https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository) for the following categories:
- Bug report
- Documentation issue report
- Feature request (just points out to the RFC repository, which can be handy for newcomers).
- Other

Here is how the "Issues" page will look like when merged into `master`:
https://github.com/alexeykostevich/issue-templates/issues/new/choose

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+### 😕 Actual Behavior
+A clear and concise description of what is happening.
+
+### 🤔 Expected Behavior
+A clear and concise description of what you expected to happen.
+
+### 🌍 Environment
+
+- Ember: 3.18
```
Comment (rwjblue) [inline .github/ISSUE_TEMPLATE/1-bug-report.md:31]: Let's not include the Ember version in the template (I'll forget to update).

Comment (alexeykostevich) [reply .github/ISSUE_TEMPLATE/1-bug-report.md]: That's a fair point. Done!

**Comment by rwjblue** [comment]
Awesome, thank you @alexeykostevich! I've added this to the agenda for the teams meeting this coming Friday, just to make sure everyone is on board.

**Review by MelSumner — COMMENTED** [review]
👍 thank you for doing this, it looks great!

**Comment by rwjblue** [comment]
Lets try it! Thanks again @alexeykostevich!

**Pull request merged**